### PR TITLE
[upstream-sync] Version 12.8.0

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

**Comment by cokiengchiara** [comment]
@giantswarm/team-halo please review so we could release prometheus upgrade? :)

File: helm/prometheus-operator-app/README.md [hunk 1]
```diff
@@ -77,16 +71,23 @@ kubectl delete crd thanosrulers.monitoring.coreos.com
 ## Upgrading Chart
 
 ```console
-# Helm 3 or 2
+# Helm
 $ helm upgrade [RELEASE_NAME] prometheus-community/kube-prometheus-stack
 ```
 
+With Helm v3, CRDs created by this chart are not updated by default and should be manually updated.
+Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions).
```
Comment (TheoBrigitte) [inline helm/prometheus-operator-app/README.md:79]: This is not true, thanks to @glitchcrab and the work from https://github.com/giantswarm/prometheus-operator-app/pull/63.

CRDs should now always be updated.

**Review by TheoBrigitte — COMMENTED** [review]
How about we first release the current version in master (0.43.2) before going with another upstream sync?

This is very hard to review. Was there any manual changes in there? Was this achieved via `script/sync_chart.sh` ? Don't we care about the patch ?

File: helm/prometheus-operator-app/README.md (hunk 1 above)
Comment (QuentinBisson) [reply helm/prometheus-operator-app/README.md:79]: This readme is synced directly from the repo. I am wondering if your sync mechanism could be proposed upstream

**Pull request closed without merging**